packages/custom-functions-metadata/test/src/test.ts

```diff
@@ -2,9 +2,10 @@
 // Licensed under the MIT license.
 
 import * as assert from "assert";
-import * as fs from "fs";
+import { readFileSync } from "fs";
 import * as mocha from "mocha";
-import { generateCustomFunctionsMetadata, IParseTreeResult, parseTree } from "../../src/generate";
+import { generateCustomFunctionsMetadata } from "../../src/generate";
```

Comment:
How come import these separately instead of from main?

Reply:
I think generally tests imported directly but not sure there's a reason to do it a different way.

Reply:
Generally, it might be preferable to import from the package if testing external exports. This ensures what you intend to export is (1) exported (2) tested. Additionally, (3) it avoids having to touch unit test files to adjust imports when package internals change (less churn).

Reply:
That point of the tests is not to test the external exports but rather the functionality.